{
    "repository": "django-import-export/django-import-export",
    "clone_url": "https://github.com/django-import-export/django-import-export.git",
    "timestamp": "2023-05-29T14:38:07.308630Z",
    "clone_success": true,
    "number of actions": 2,
    "number_of_test_actions": 1,
    "actions_successful": false,
    "actions_stdout": "[django-import-export CI/test] \ud83d\ude80  Start image=crawlergpt:latest\n[django-import-export CI/test]   \ud83d\udc33  docker pull image=crawlergpt:latest platform= username= forcePull=false\n[django-import-export CI/test]   \ud83d\udc33  docker create image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[django-import-export CI/test]   \ud83d\udc33  docker run image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[django-import-export CI/test]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1000 /tmp/de65f406-fe28-11ed-a890-af2cc187fc11/django-import-export-django-import-export] user=0 workdir=\n[django-import-export CI/test]   \u2601  git clone 'https://github.com/actions/setup-python' # ref=v4\n[django-import-export CI/test] \ud83e\uddea  Matrix: map[python-version:3.7]\n[django-import-export CI/test] \u2b50 Run Main Set up MySQL\n[django-import-export CI/test]   \ud83d\udc33  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/0] user= workdir=\n[django-import-export CI/test]   | sudo: /etc/init.d/mysql: command not found\n[django-import-export CI/test]   \u274c  Failure - Main Set up MySQL\n[django-import-export CI/test] exitcode '1': failure\n[django-import-export CI/test] \ud83c\udfc1  Job failed\n",
    "actions_stderr": "Error: Job 'test' failed\n"
}